plugin/src/main/java/com/denizenscript/denizen/objects/properties/entity/EntityOnBack.java

```diff
@@ -0,0 +1,41 @@
+package com.denizenscript.denizen.objects.properties.entity;
+
+import com.denizenscript.denizen.objects.EntityTag;
+import com.denizenscript.denizencore.objects.Mechanism;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import org.bukkit.entity.Panda;
+
+public class EntityOnBack extends EntityProperty<ElementTag> {
+
+    // <--[property]
+    // @object EntityTag
+    // @name is_on_back
+    // @input ElementTag(Boolean)
+    // @description
+    // Controls whether a panda is on its back.
+
+    public static boolean describes(EntityTag entity) {
+        return entity.getBukkitEntity() instanceof Panda;
+    }
+
```

Comment:
these properties should probably all have `false` set as a nullable default to avoid cluttering the property output too much